app/views/organizations/_details.html.erb

```diff
@@ -20,42 +20,48 @@
                 </p>
               </div>
               <div class="mb-4">
-                <h3>Repackage Essentials?</h3>
+                <h3 class='font-bold'>NDBN Membership ID</h3>
+                <p>
+                  <%= @organization.ndbn_member&.full_name || "Not Defined"%>
```

Comment:
Maybe "Not Provided" instead of "Not Defined"?

Reply:
+